src/aeso_aci.erl

```diff
@@ -83,7 +83,7 @@ from_typed_ast(Type, TypedAst) ->
         string -> do_render_aci_json(JArray)
     end.
 
-encode_contract(Contract = {Head, _, {con, _, Name}, _}) when ?IS_CONTRACT_HEAD(Head) ->
```

Comment:
I think we are soon approaching the point where this should become a record, something to consider for the _next_ addition 😅